test/client_test.rb

```diff
@@ -129,27 +129,24 @@ class ClientTest < TinyTds::TestCase
       end
     end
 
-    it 'must run this test to prove we account for dropped connections' do
-      skip
+    it 'raises TinyTds exception with sql batch timeout due to network failure' do
+      skip if ENV['CI'] && ENV['APPVEYOR_BUILD_FOLDER'] # only CI using docker
       begin
-        client = new_connection :login_timeout => 2, :timeout => 2
+        client = new_connection timeout: 2
         assert_client_works(client)
-        STDOUT.puts "Disconnect network!"
-        sleep 10
-        STDOUT.puts "This should not get stuck past 6 seconds!"
+        docker_container('pause')
         action = lambda { client.execute('SELECT 1 as [one]').each }
         assert_raise_tinytds_error(action) do |e|
           assert_equal 20003, e.db_error_number
           assert_equal 6, e.severity
           assert_match %r{timed out}i, e.message, 'ignore if non-english test run'
         end
       ensure
-        STDOUT.puts "Reconnect network!"
-        sleep 10
+        docker_container('unpause')
         action = lambda { client.execute('SELECT 1 as [one]').each }
         assert_raise_tinytds_error(action) do |e|
           assert_equal 20047, e.db_error_number
-          assert_equal 1, e.severity
+          assert_equal 9, e.severity
```

Comment:
Interesting that this is changed, because for me the test is currently failing because the error severity actually is `1`. Now I'm surprised/worried if you and I are getting different severity levels for the same error.

Reply:
I wonder if this has to do with different versions of freetds? I'm currently running `1.2.3` on my machine. Also, it's worth noting that this test on Travis CI is passing which looks to be running `1.1.24`. If you try out one of those versions does the severity change?

Reply:
Hm, that is definitely a (sad) possibility. I'm running v1.00.112. I'll try and upgrade and see if that changes things.

Reply:
This commit would indicate it is in fact the underlying freetds version: https://github.com/FreeTDS/freetds/commit/66423608f405d5714b047a67652d8a6015094f29. Not sure what to do here...maybe a comment in the test explaining that the test could fail due to severity but that's ok?

Reply:
We have a couple options

1. just change it (preferably in a commit by itself with a clear explanation & link over to that freetds commit)
2. do 1, plus changing the assertion to `assert_includes [1,9], e.severity`
3. do 1, but also bump the minimum supported version of FreeTDS

I vote option 2 because it's maximally clear about what could happen for various TinyTDS users. I don't think this particular situation warrants option 3.

Reply:
I agree option 2 is the best here. I'll update. Thanks!